platform/logging/appenders/console/ConsoleAppender.ts

```diff
@@ -0,0 +1,14 @@
+import { LogRecord } from '../../LogRecord';
+import { BaseAppender } from '../base/BaseAppender';
+import { ConsoleAppenderConfig } from './ConsoleAppenderConfig';
+
+export class ConsoleAppender extends BaseAppender {
+  constructor(protected readonly config: ConsoleAppenderConfig) {
+    super(config)
+  }
+
+  append(record: LogRecord) {
+    super.append(record);
+    console.log(this.logRecordToFormattedString(record));
```

Comment:
I was thinking something along the lines of current Kibana:

<img width="726" alt="screen shot 2017-07-17 at 11 51 32" src="https://user-images.githubusercontent.com/5259/28263250-546412a0-6ae6-11e7-96fb-ab64689b9619.png">

It makes it way easier to see `error` and `warn`, for example, plus the context.